extension/sumologicextension/extension.go

```diff
@@ -709,21 +709,59 @@ var sumoAppProcesses = map[string]string{
 	"redis":                 "redis",
 	"tomcat":                "tomcat",
 	"kafka-server-start.sh": "kafka", // Need to test this, most common shell wrapper.
+	"redis-server":          "redis",
+	"mongod":                "mongodb",
+	"cassandra":             "cassandra",
+	"jmx":                   "jmx",
+	"activemq":              "activemq",
+	"memcached":             "memcached",
+	"haproxy":               "haproxy",
+	"dockerd":               "docker-ce", // docker engine, for when process runs natively
+	"com.docker.backend":    "docker-ce", // docker daemon runs on a VM in Docker Desktop, process doesn't show on mac
+	"sqlservr":              "mssql",     // linux SQL Server process
 }
 
 func filteredProcessList() ([]string, error) {
 	var pl []string
 
-	p, err := ps.Processes()
+	processes, err := process.Processes()
 	if err != nil {
 		return pl, err
 	}
 
-	for _, v := range p {
-		e := strings.ToLower(v.Executable())
+	for _, v := range processes {
+		// Get process executable name
+		e, err := v.Name()
+		if err != nil {
+			return nil, fmt.Errorf("Error getting executable name: %w", err)
+		}
+		e = strings.ToLower(e)
+
 		if a, i := sumoAppProcesses[e]; i {
 			pl = append(pl, a)
 		}
+
+		// handling for Docker Desktop
+		if e == "com.docker.backend" {
+			pl = append(pl, "docker-ce")
```

Comment:
Might be a good idea to define constants for  strings like "com.docker.backend"

and also refer to the "docker-ce" from the `sumoAppProcesses` map